examples/server/server.cpp

```diff
@@ -430,6 +446,39 @@ int main(int argc, const char ** argv) {
     svr.reset(new httplib::Server());
 #endif
 
+    std::unordered_map<std::string, std::string> model_map = {};
+    const std::string model_path = args.get_string_param("--model-path");
```

Comment:
the arg description should be updated to describe this included functionality and its limitations. E.G. `"(REQUIRED) The local path of the gguf model file for Parler TTS mini or large v1, Dia, or Kokoro."` should be changed to something like `""(REQUIRED) The local path of the gguf model file or a directory containing only gguf model files for Parler TTS mini or large v1, Dia, or Kokoro."`.